src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -121,6 +126,17 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 })
             );
 
+            // TODO Is this the best solution to the routing - controllers can be accessed on both "/tools" and "/api"
```

Comment:
I expect this is a converszation to be had with @jamescutts - happy that this PR shouldn't be blocked in the meantime though